src/screens/AdminPortal/ManageTag/editModal/EditUserTagModal.tsx

```
@@ -18,28 +18,24 @@
- * - tCommon: tCommon
  *
  * @returns The rendered edit user tag modal.
  */
 // translation-check-keyPrefix: manageTag
-import type { TFunction } from 'i18next';
 import type { FormEvent } from 'react';
 import React, { useEffect, useRef, useState } from 'react';
-import Button from 'shared-components/Button';
-import { CRUDModalTemplate } from 'shared-components/CRUDModalTemplate/CRUDModalTemplate';
-import { FormTextField } from 'shared-components/FormFieldGroup/FormFieldGroup';
+import { Button, Form } from 'react-bootstrap';
```

ESLint

[error] 'Button' import from 'react-bootstrap' is restricted. Direct imports of Button from react-bootstrap are not allowed. Use the shared Button component from src/shared-components/Button/ instead.

(no-restricted-imports)